package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vets-json-schema",
-  "version": "6.6.3",
+  "version": "6.7.0",
```

Comment:
Please double-check the version in `master` before merging? There are several PRs open right now. Thank you!!